disnake/ext/tasks/__init__.py

```diff
@@ -73,31 +100,35 @@ def cancel(self) -> None:
         self.future.cancel()
 
 
-class Loop(Generic[LF]):
+class Loop(Generic[CoroP]):
     """A background task helper that abstracts the loop and reconnection logic for you.
 
     The main interface to create this is through :func:`loop`.
     """
 
     def __init__(
         self,
-        coro: LF,
-        seconds: float,
-        hours: float,
-        minutes: float,
-        time: Union[datetime.time, Sequence[datetime.time]],
-        count: Optional[int],
-        reconnect: bool,
-        loop: asyncio.AbstractEventLoop,
+        coro: Coro[CoroP],
+        seconds: float = 0,
+        minutes: float = 0,
+        hours: float = 0,
+        time: Union[datetime.time, Sequence[datetime.time]] = MISSING,
+        count: Optional[int] = None,
+        reconnect: bool = True,
+        loop: asyncio.AbstractEventLoop = MISSING,
```

Comment:
might be worth making these keyword only arguments in this same pr, since they're currently being reordered which is already breaking